src/popup/index.tsx

```diff
@@ -17,7 +17,6 @@ root.render(
             className='popup'
             style={{
                 position: 'relative',
-                minHeight: '100vh',
```

Comment:
why remove this line?

Reply:
Now the normal height cannot be restored after pulling down the input area